eggplant/market/models/cart.py

```diff
@@ -1,6 +1,10 @@
 from decimal import Decimal
 
 from django.db import models, transaction
+from .payment import Payment, PaymentItem
+from django.db.models import Sum, F, Count, Max
+from django.core.exceptions import ValidationError
+from moneyed import Money
```

Comment:
When installed, it will fire, but you have to run `pre-commit install`, after that it lints changed files before committing.